cmake/helpers/CheckDependentLibraries.cmake

```diff
@@ -195,7 +195,11 @@ if (GDAL_USE_CRYPTOPP)
   option(CRYPTOPP_USE_ONLY_CRYPTODLL_ALG "Use Only cryptoDLL alg. only work on dynamic DLL" OFF)
 endif ()
 
-find_package(PROJ 6.0 REQUIRED)
+# First check with CMake config files, and then fallback to the FindPROJ module.
+find_package(PROJ 8.0 CONFIG)
+if (NOT TARGET PROJ::proj)
```

Comment:
The key indicator of success is `PROJ_FOUND`.

Reply:
done